### DISCUSSION TITLE
Text in q-card-section will not wrap

### DISCUSSION THREAD

**Discussion by Dirk-** [quasarframework/quasar#9571] [General - Components / Directives / etc] [answered]
I'd like to have two QCards in an expandable row of a QTable. The first one should display an image, the second one a title and a text below. I got it this far:

<img width="996" alt="Bildschirmfoto 2021-06-05 um 14 45 59" src="https://user-images.githubusercontent.com/7225580/120892405-69b14880-c60e-11eb-8e89-fe5285f83b11.png">

As you can see, the text in the second Card does not wrap. My template does not look much different from the Quasar docs:

        <q-tr v-show="props.expand" :props="props">
            <q-td colspan="100%">
              <div class="row q-gutter-md">
                <q-card class="my-card">
                  <q-card-section>
                  <q-img
                    style="min-width: 200px"
                    src="https://cdn.quasar.dev/img/parallax2.jpg"
                  >
                  </q-img>
                  </q-card-section>
                </q-card>
                <q-card class="col my-card">
                  <q-card-section class="text-subtitle2">
                    Notiz
                  </q-card-section>

                  <q-card-section>
                    {{ props.row.notiz }}
                  </q-card-section>
                </q-card>
              </div>
            </q-td>
          </q-tr>

I'd like the text to stay inside the card. What am I doing wrong?

**Comment by ontwikkelfabriek** [comment, marked as answer]
@Dirk- 
Please create a codepen.io to demonstrate your problem so we can provide better support.

**Reply by Dirk-** [reply]
Here is the pen: https://codepen.io/Dirk-/pen/gOmzRzP

**Reply by ontwikkelfabriek** [reply]
@Dirk- 

Here you go:

```javascript
<q-card-section style="white-space: normal">
   {{ props.row.name }}: {{ lorem }}
</q-card-section>
```

**Reply by Dirk-** [reply]
Hey, this works! Great, thank you!
